cfme/web_ui/__init__.py

```diff
@@ -2061,6 +2061,8 @@ def _find_in_tree(t, p=None):
 
 @fill.method((BootstrapTreeview, Sequence))
 def _fill_bstree_seq(tree, values):
+    if not values:
```

Comment:
This is where I considered setting the default for `Role.product_features` to `[(['Everything'], true)]` instead of `[]` to match the UI default and prevent the exception.